projects/urls.py

```diff
@@ -9,5 +9,6 @@
     path("admin/", admin.site.urls),
     path("media_server/", include("openedx_learning.contrib.media_server.urls")),
     path("rest_api/", include("openedx_learning.rest_api.urls")),
+    path("tagging/rest_api/", include("openedx_tagging.core.tagging.urls", namespace="oel_tagging")),
```

Comment:
Ah, I steered you wrong here.. you don't have to specify the namespace here or the app_name above. Since the tagging app already has a name, its namespace is used by default:

```suggestion
    path("tagging/rest_api/", include("openedx_tagging.core.tagging.urls")),
```